c10/util/int128.h

```diff
@@ -0,0 +1,143 @@
+#pragma once
+
+/*
+ * A very basic uint128 class, based on absl version, used as the basis for
+ *128-bit hashing for the lazy tensor project. If using this for other purposes,
+ *it might make sense to replace this with a more complete implementation.
```

Comment:
Nit: A space is needed after *.

Reply:
lol.  really?  I had to delete those spaces to pass the linter (!)

Reply:
Then, never mind.